cmd/virtual-workspaces/command/cmd.go

```diff
@@ -55,8 +55,8 @@ func NewCommand(ctx context.Context, errout io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "workspaces",
-		Short: "Launch workspaces virtual workspace apiserver",
-		Long:  "Start a virtual workspace apiserver to managing personal, shared or organization workspaces",
+		Short: "Launch virtual workspace apiservers",
+		Long:  "Start the root virtual workspace apiserver to enable virtual workspace management.",
```

Comment:
@sttts updated here -- happy to do more iteration if this feels under-specified or inaccurate

Reply:
It's not super important clearly. If you run this command manually, you know what you are doing.